src/canvascontext.js

```diff
@@ -98,7 +120,10 @@ export class CanvasContext {
   }
 
   resize(width, height) {
-    return this.vexFlowCanvasContext.resize(parseInt(width, 10), parseInt(height, 10));
+    width = parseInt(width, 10);
+    height = parseInt(height, 10);
+    [width, height] = this.CheckCanvasDimensionsForBrowserLimit(width, height);
```

Comment:
You can move the `parseInt` into the above `SanitizeCanvasDims` (which is a better place for it.)

Reply:
I assume like this:
`[width, height] = this.SanitizeCanvasDims(parseInt(width, 10), parseInt(height, 10));`
becuse moving it into the `SanitizeCanvasDims` function would mean i 'd need to call parseInt again for width and height for `this.vexFlowCanvasContext.resize(width, height);`, which would be duplicated.

Reply:
Okay, that's fine, thanks.